### PR TITLE
docs: setup MySQL / SQLite, an over simplified guide introduction to …

### PR DESCRIPTION
**Pull request by cyjake**
…quick start

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 [![NPM Downloads](https://img.shields.io/npm/dm/leoric.svg?style=flat)](https://www.npmjs.com/package/leoric)
 [![NPM Version](http://img.shields.io/npm/v/leoric.svg?style=flat)](https://www.npmjs.com/package/leoric)
 [![Build Status](https://travis-ci.org/cyjake/leoric.svg)](https://travis-ci.org/cyjake/leoric)
-[![Coverage Status](https://coveralls.io/repos/github/cyjake/leoric/badge.svg?branch=master)](https://coveralls.io/github/cyjake/leoric?branch=master)
```
Comment (cyjake) [inline Readme.md:7]: https://coveralls.io/ still down for maintenance 😕

File: docs/_layouts/en.html [hunk 2]
```diff
@@ -11,7 +11,7 @@
           <a href="{{ '/' | relative_url }}">Home</a>
         </li>
         <li class="nav-item dropdown-trigger">
-          <a href="{{ '/basics' | relative_url }}">Guides</a>
+          <a href="{{ '/guides' | relative_url }}">Guides</a>
```
Comment (cyjake) [inline docs/_layouts/en.html:14]: still not 100% sure this is a good idea, trying to provide a compact guide with an abstract project to give an simpler overview about Leoric features.

Comment (JimmyDaddy) [reply docs/_layouts/en.html:14]: I think an example project more useful than a good document☺️

Comment (cyjake) [reply docs/_layouts/en.html:14]: agree, will expand this documentation into an example project :-)

**Review by JimmyDaddy — APPROVED** [review]

**Pull request merged**